Source/WebCore/Modules/webdatabase/SQLTransactionStateMachine.h

```diff
@@ -51,8 +51,8 @@ class SQLTransactionStateMachine {
     // s_sizeOfStateAuditTrail states that the state machine enters. The audit
     // trail is updated before entering each state. This is for debugging use
     // only.
-    static const int s_sizeOfStateAuditTrail = 20;
-    int m_nextStateAuditEntry;
+    static constexpr unsigned s_sizeOfStateAuditTrail = 20U;
+    unsigned m_nextStateAuditEntry;
```

Comment:
I don't think you actually need to change this? Unless you're trying to avoid a -Wsign-compare warning or there is some other clear benefit, I wouldn't change integer types because you risk introducing bugs by mistake. Just observing that the value should never be negative is not a good reason to use an unsigned type; I would only use unsigned types for bit manipulations or when some API requires unsigned. See the [arithmetic recommendations](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#arithmetic) from the core guidelines. WebKit doesn't really follow these recommendations, but maybe we should in new code. And we certainly should if changing integer types without a strong reason to do so.